docs/readthedocs/references.rst

```diff
@@ -0,0 +1,21 @@
+************
+References
+************
+
+Publications
+===============
+
+The following publications discuss SIMONA and implementation details as well as outcomes where SIMONA have been used:
+
+.. rubric:: References about SIMONA
+
+.. bibliography:: _static/bibliography/bibAboutSimona.bib
+   :list: enumerated
+   :all:
+
+
+.. rubric:: References
+.. bibliography:: _static/bibliography/bibtexAll.bib
+   :all:
+
+
```

Comment:
might make sense to go with `.md` here

Reply:
It would, but the bibtex integration runs through rst and that is a bit of a hassle. For that reason I would like to keep rst here and also at the index page since someone finds time to tackle this adequate.